superset/utils/database.py

```diff
@@ -65,11 +65,7 @@ def get_or_create_db(
 
 
 def get_example_database() -> Database:
-    db_uri = (
-        current_app.config.get("SQLALCHEMY_EXAMPLES_URI")
-        or current_app.config["SQLALCHEMY_DATABASE_URI"]
-    )
-    return get_or_create_db("examples", db_uri)
+    return get_or_create_db("examples", current_app.config["SQLALCHEMY_EXAMPLES_URI"])
```

Comment:
Regarding https://github.com/apache/superset/discussions/23399, should we have the examples DB have DML on and with file uploads enabled?

Reply:
left a comment on #23399, no strong opinions about allowing DML/upload on the examples db, yet I do see this as a readonly db